::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2018 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sat, 2 Jun 2018 21:47:46 +0200
Subject: [PATCH] network.interface: Allow gw for ovs_{port,bridge}

Previous behavior was to silently ignore interface.gateway for
ovs_port and ovs_bridge interface types, rework conditions to
allow it just like for regular interfaces.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 linux/network/interface.sls | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/linux/network/interface.sls b/linux/network/interface.sls
index f2691a5..94300a0 100644
--- a/linux/network/interface.sls
+++ b/linux/network/interface.sls
@@ -283,6 +283,8 @@ remove_interface_{{ network }}_line2:

 {%- endfor %}

+{%- endif %}
+
 {%- if interface.gateway is defined %}

 linux_system_network:
@@ -301,8 +303,6 @@ linux_system_network:

 {%- endif %}

-{%- endif %}
-
 {%- if interface.wireless is defined %}

 {%- if grains.os_family == 'Arch' %}
